src/LibFLExBridge-ChorusPlugin/FLExProjectSplitter.cs

```diff
@@ -36,22 +36,22 @@ namespace LibFLExBridgeChorusPlugin
 	///		B. One file for the model version
 	///		C. Various files for the CmObject data.
 	/// </summary>
-	public static class FLExProjectSplitter
+	internal class FLExProjectSplitter: IProjectSplitter
```

Comment:
I would have wanted a conversation about this change if I'd seen it back when it happened. I can't track down any unit tests related to this yet. I have trouble imagining what kind of mocking makes this code more testable then when it was static methods. :-(

Reply:
Hmm - this is strange. GitHub shows the diff is backwards. In the latest version it actually changed from 

    internal class FLExProjectSplitter: IProjectSplitter

to

    internal static class FLExProjectSplitter

Looks like GitHub has a bug here: if I got to the "Files changed" tab and scroll down to the [file](https://github.com/sillsdev/flexbridge/pull/173/files#diff-2bba92d1e77a906cb22ac368aafc2fb1) (or search for FLExProjectSplitter) and then click on "Load diff" it shows correctly. 
If however, I go to the "Conversations" tab, scroll down to your comment, click View Changes and then use Jump to from the menu to get to the file it shows the wrong changes. Ahh, but that's probably because for whatever reason it shows a subset of the commits (top shows: "changes from 23 commits" instead of 115).

What you looked at was an intermediate step where I thought we'd need separate splitters for FLEx and LfMerge. Later it turned out that we can use the same, so I changed it back to a static method.